bee-node/src/plugins/version_checker.rs

```diff
@@ -31,7 +40,11 @@ impl<N: Node> Worker<N> for VersionChecker {
             );
 
             while ticker.next().await.is_some() {
-                // TODO
+                match is_latest(&client).await {
+                    Ok(false) => warn!("A new version has been released. Please update the node at https://github.com/iotaledger/bee/releases"),
+                    Err(e) => error!("error while checking for new update. {:?}", e),
```

Comment:
I suggest changing `.` (point) to `:` (colon).

Reply:
+1

Reply:
Same for all errors btw